src/eko/evolution_operator/__init__.py

```diff
@@ -313,34 +313,22 @@ def grid_size(self):
         """Return the grid size."""
         return self.int_disp.xgrid.size
 
-    def sv_exponentiated_shift(self, q2):
-        """Compute shifted renormalization scale.
-
-        Parameters
-        ----------
-        q2 : float
-            factorization scale
-
-        Returns
-        -------
-        float
-            renormalization scale
-        """
-        if self.sv_mode == sv.Modes.exponentiated:
-            return q2 / self.xif2
-        return q2
-
     @property
     def a_s(self):
         """Return the computed values for :math:`a_s`."""
         sc = self.managers["strong_coupling"]
         a0 = sc.a_s(
-            self.sv_exponentiated_shift(self.q2_from),
+            self.q2_from * self.xif2,
```

Comment:
I like that there is no exception any longer, but of course it is good only if it works for both...

Reply:
it won't work for both ... (as said on scheme B I've nothing decided yet)